source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -47,6 +50,34 @@ class EvaluationContext {
     set $data(value: any) {
         this._$data = value;
     }
+
+    generateVersionJson() {
```

Comment:
Add a unit test if possible

Reply:
@licanhua I added a basic unit test that fails if major, minor, or patch are not parsed correctly. The only other test I can think of is to directly check if all parts of the JSON are correct, but I believe that would require us to update the test whenever the version changes.